genestack_client/version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = '0.5.2a1'
+__version__ = '0.5.2a2'
```

Comment:
I think that the `version` should become `0.6.0`. Consult @genestack-solomatin about that.